### DISCUSSION TITLE
Instances of containers

### DISCUSSION THREAD

**Discussion by Gitgeridoo** [jgraph/drawio#4433] [Ideas]
Hello,

In more complex flowcharts (process diagrams) it's quite common for certain parts to be the same among different process flows.

Currently, such parts can be put in a container and then copied to other flows. However, when there are changes, one has to make that change in every occurrence separately. This is quite cumbersome and prone to mistakes.

That's why it would be useful if it would be possible to create container instances. All edits in one of the instances would automatically be applied to all of them.

An example of how this works can be seen in Adobe Illustrator with the use of Symbols.

Kind regards,
Ron van der Lee.

**Comment by 0Grit** [comment]
Possible duplicate of https://github.com/jgraph/drawio/discussions/4359 ?
